### DISCUSSION TITLE
Add an option to switch from 2014-2015 or 2016 header menu button

### DISCUSSION THREAD

**Discussion by Xammand** [Rehike/Rehike#580] [Ideas]
### Describe the feature

This is very minor but since there are some options in Rehike that turns things into 2014-2015 Youtube stuff (Ex: Upload button with text, no 'views' text for views, 2014 or 15 player, etc.) I thought it would be nice to also have an option to be able to switch from 2014-2015 or 2016 Youtube header menu button. Just so it can be more accurate to 2014-2015 YouTube if one uses all those other options on Rehike for that version of Youtube. Yes, I use all the 2014-2015 YouTube options on Rehike.

2014-2015 YouTube header menu button: ![Screenshot_166](https://github.com/Rehike/Rehike/assets/84749558/07a8f7ba-9200-4274-b17d-cbbf94d7ee49)
2016 YouTube header menu button (The one used on Rehike): ![Screenshot_167](https://github.com/Rehike/Rehike/assets/84749558/0ecc9385-b1aa-42ef-9358-4546c1620171)

**Comment by kawapure** [comment]
There are not plans to support 2014 features (right-aligned guide button as in this issue, watch7 layout, etc.) in Rehike officially. I recommend using this userstyle instead: https://userstyles.world/style/9427/early-2014-youtube-for-rehike

**Reply by Xammand** [reply]
Sorry for the late reply, the userstyle has worked well for me. Thanks.
